packages/react-scripts/config/webpack.config.dev.js

```diff
@@ -243,41 +268,43 @@ module.exports = {
           // in development "style" loader enables hot editing of CSS.
           // By default we support CSS Modules with the extension .module.css
           {
-            test: /\.css$/,
-            exclude: /\.module\.css$/,
-            use: [
-              require.resolve('style-loader'),
-              {
-                loader: require.resolve('css-loader'),
-                options: {
-                  importLoaders: 1,
-                },
-              },
-              {
-                loader: require.resolve('postcss-loader'),
-                options: postCSSLoaderOptions,
-              },
-            ],
+            test: cssRegex,
+            exclude: cssModuleRegex,
+            use: getStyleLoaders({
+              importLoaders: 1,
+            }),
           },
           // Adds support for CSS Modules (https://github.com/css-modules/css-modules)
           // using the extension .module.css
           {
-            test: /\.module\.css$/,
-            use: [
-              require.resolve('style-loader'),
-              {
-                loader: require.resolve('css-loader'),
-                options: {
-                  importLoaders: 1,
-                  modules: true,
-                  getLocalIdent: getCSSModuleLocalIdent,
-                },
-              },
+            test: cssModuleRegex,
+            use: getStyleLoaders({
+              importLoaders: 1,
+              modules: true,
+              getLocalIdent: getCSSModuleLocalIdent,
+            }),
+          },
+          // Opt-in support for SASS (using .scss or .sass extensions).
+          // Chains the sass-loader with the css-loader and the style-loader
+          // to immediately apply all styles to the DOM.
+          // By default we support SASS Modules with the
+          // extensions .module.scss or .module.sass
+          {
+            test: sassRegex,
+            exclude: sassModuleRegex,
+            use: getStyleLoaders({}, 'sass-loader'),
```

Comment:
@Timer, as far as I understood `importLoaders` here would allow e.g. to import SASS into CSS files, which is not common, and as SASS already handles imports then this wouldn't be needed. Could anyone please confirm that just to see if I didn't misunderstood? https://github.com/webpack-contrib/css-loader/issues/228

Reply:
I could see someone importing a sass file in a css file; if there's no downsides/caveats we may as well add it and a corresponding test.

Reply:
In fact I've just tried importing a sass into a css and it works, so I wonder what's the use for `importLoaders` here?... Anyone with better understanding on webpack to give us a hint?

Reply:
here is the original issue and attached diff adding this option, not sure if it'll help: https://github.com/webpack-contrib/css-loader/issues/21

Reply:
Updated using `{importLoaders: 2}` according to `css-loader` docs: https://github.com/webpack-contrib/css-loader#importloaders